src/gridcoin/scraper/scraper_net.h

```diff
@@ -164,6 +164,7 @@ class CScraperManifest
     bool bCheckedAuthorized;
 
 public: /* public methods */
+    virtual ~CScraperManifest();
```

Comment:
The virtuality comes from `CSplitBob`, so what I think you want is `virtual ~CSplitBob() = default;` and it should be good to go.

Reply:
Ah. But I just looked. The CSplitBlob destructor already has a concrete implementation. I think all we have to do is add "virtual" to the header definition. I am trying that now.